x-pack/plugins/security_solution/public/cases/components/case_action_bar/helpers.ts

```diff
@@ -21,4 +22,4 @@ export const getStatusDate = (theCase: Case): string | null => {
   return null;
 };
 
-export const getStatusTitle = (status: CaseStatuses) => statuses[status].actionBar.title;
+export const getStatusTitle = (status: CaseStatuses) => get(statuses, `${status}.actionBar.title`);
```

Comment:
nit: Now that the `actionBar` is optional the function should return `string | undefined`

Reply:
The input status here does not include `all`,  and we do have actionBar for `open`, `in-progress`, and `closed`, I think we can put the return type as string?